app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ActiveRecord::Base
+  has_secure_password validations: false
+
+  validates :password, length: {minimum: 3}
+  validates :email, length: {minimum: 5}
```

Comment:
You can combine this line and the next one, since they are regarding the same attribute.